src/BookParser.hs

```diff
@@ -72,8 +72,9 @@ renderWarning = \case
 
 matchString :: String -> String -> Maybe String
 matchString model actual = case splitAt (length model) actual of
-  (model', rest) | model == model' -> Just rest
-  _ -> Nothing
+  (model', rest)
+    | model == model' -> Just rest
```

Comment:
meh :unamused:

Reply:
This alignment is not done by `stylish-haskell`. I did some part because it puts `->` under `|`, not under corresponding `->`. But I understood that extra alignment is not what you want because you want to minimize git diffs...